### PR TITLE
RHEL8: README.Linux.md update on 6X_STABLE

### PR DESCRIPTION
**Pull request by krishnanmano**
Updating RHEL8 installation steps on README.Linux.md, as few additional RHEL8 specific steps needs to be updated.

Co-authored-by: Piyush Chandwadkar pchandwadkar@vmware.com

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: README.linux.md [hunk 1]
```diff
@@ -23,16 +23,42 @@
...
+    sudo ln -s /usr/bin/python2.7 /usr/bin/python
+    ```
+
+  - Install dependencies using README.CentOS.bash script.
+
+    ```bash
+    ./README.CentOS.bash
+    ```
+
+- Note: GPDB 6X_STABLE requires `xerces 3.1` or [`gp-xerces`](https://github.com/greenplum-db/gp-xerces).
```
Comment (sumitbasumallick) [inline README.linux.md:60]: There is `xerces` section explicitly present after `Installing dependencies (for Linux developers)` for 6X_STABLE. We better remove this Note.

File: README.linux.md [hunk 2]
```diff
@@ -23,16 +23,42 @@
...
+    ```
+
+  - Install dependencies using README.CentOS.bash script.
+
+    ```bash
+    ./README.CentOS.bash
+    ```
+
+- Note: GPDB 6X_STABLE requires `xerces 3.1` or [`gp-xerces`](https://github.com/greenplum-db/gp-xerces). 
+  If other version of `xerces` installed in the System, remove it and install `xerces 3.1` or [`gp-xerces`](https://github.com/greenplum-db/gp-xerces).
```
Comment (sumitbasumallick) [inline README.linux.md:61]: Does other version of `xerces` getting automatically installed?

File: README.linux.md [hunk 3]
```diff
@@ -58,70 +84,70 @@ Use dependency script for CentOS.
...
+    * soft nproc 131072
+    * hard nproc 131072
+
+    EOF'
+
+    sudo bash -c 'cat >> /etc/ld.so.conf <<-EOF
+    /usr/local/lib
+
+    EOF'
+    ```
```
Comment (sumitbasumallick) [inline README.linux.md:153]: Is there any change in Ubuntu section proposed?

File: README.linux.md (hunk 2 above)
Comment (krishnanmano) [reply README.linux.md:61]: Thanks Sumit, for the comments. Other version of xerces will not be installed, as installation of `xerces-c-devel` removed from README.CentOS.bash.

File: README.linux.md (hunk 1 above)
Comment (krishnanmano) [reply README.linux.md:60]: Thanks Sumit, for the comments. Sure will remove the note.

File: README.linux.md (hunk 3 above)
Comment (krishnanmano) [reply README.linux.md:153]: Thanks Sumit, for the comments. Just changed the indentation on few sections.

**Review by sumitbasumallick — APPROVED** [review]
+1

**Pull request merged**